build/lib/crystal_toolkit/__init__.py

```diff
@@ -0,0 +1,94 @@
+import json
```

Comment:
Files in `build/*` should also be ignored, not committed. Might be worth purging these files from `git` history since they're quite large.